libs/Wallet/include/Status/Wallet/WalletController.h

```diff
@@ -38,6 +38,7 @@ class WalletController: public QObject
     /// \todo consider if complex approach of keeping ownership here and enforcing a unique instance
     ///       or not reusing the account list and make it singleton are better options
     Q_INVOKABLE [[nodiscard]] Status::Wallet::NewWalletAccountController* createNewWalletAccountController() const;
+    Q_INVOKABLE [[nodiscard]] Status::Wallet::SavedAddressesController* createSavedAddressesController() const;
```

Comment:
It could be slightly shorter:
```
Q_INVOKABLE [[nodiscard]] SavedAddressesController* createSavedAddressesController() const;
```

Reply:
For QML calls better to use full namespace, clang-tidy also checks it as warning